### ISSUE TITLE
Source code was not found in ios 

### ISSUE THREAD

**Issue by sawaca96** [getsentry/sentry-capacitor#206]
### Environment

vue3 + capacitor 

```
 "@sentry/capacitor": "^0.9.0",
 "@sentry/tracing": "^7.9.0",
 "@sentry/vue": "^7.9.0",
```

### Steps to Reproduce

```js
import { Integrations } from '@sentry/tracing';
import * as CapacitorSentry from '@sentry/capacitor';
import * as VueSentry from '@sentry/vue';

initSentry(app, router) {
  CapacitorSentry.init(
    {
      dsn: process.env.VUE_APP_SENTRY_DSN,
      integrations: new Integrations.BrowserTracing({
        routingInstrumentation: VueSentry.vueRouterInstrumentation(router),
      }),
      tracesSampleRate: 0.1,
      environment: process.env.VUE_APP_SENTRY_ENV,
      enabled: process.env.NODE_ENV === 'production',
      release: process.env.VUE_APP_VERSION,
      dist: process.env.VUE_APP_VERSION,
      logErrors: process.env.NODE_ENV === 'production' ? false : true,
      enableNative: process.env.VUE_APP_PLATFORM !== 'web',
      trackComponents: true,
    },
    (options) => VueSentry.init({ ...options, app })
  );
}

import { createApp } from 'vue';
import router from './router';
initSentry(app, router);

```

and upload source-map using sentry-cli 

<details>
<summary> uploaded source code </summary>

<img width="1182" alt="image" src="https://user-images.githubusercontent.com/49309322/183360598-fa2c2f63-54d4-4b16-ad99-21631dfc8315.png">

</details>

### Expected Result

Same result for ios and android. and sentry issues show correct information including component info

### Actual Result

ios has error `Source code was not found` 

<details>
<summary>ios sentry issue</summary>

![image](https://user-images.githubusercontent.com/49309322/183363688-4ca387bb-fc5d-4e26-8209-1f62f65c2f01.png)
![image](https://user-images.githubusercontent.com/49309322/183363848-83a1b5c9-0222-4ece-ae9c-8a48377097ff.png)

</details>

<details>
<summary>android issue</summary>

![image](https://user-images.githubusercontent.com/49309322/183365762-d36f6d0b-49d5-4ee6-8162-90616dd5f078.png)
![image](https://user-images.githubusercontent.com/49309322/183365937-78605228-2e10-4759-ad77-2c8b4435863a.png)

</details>

**Comment by lucas-zimerman** [comment]
Thanks for opening the issue :D 
I'll investigate it and give a response once I conclude the investigation.

Also, are both Android and iOS apps with the same release number or are they different?

**Comment by sawaca96** [comment]
use same release

**Comment by lucas-zimerman** [comment]
I wasn't exactly able to reproduce it, would you mind sharing a screenshot from the raw stacktrace from that issue?
![image](https://user-images.githubusercontent.com/8229322/187207929-f394c549-1de9-4fe8-b690-3a91027d1949.png)

Also, if possible, a repro (I found another issue that may or may not be related with the Vue stack trace)

**Comment by sawaca96** [comment]
Sorry for late 🙏
![image](https://user-images.githubusercontent.com/49309322/205527065-c8b74400-2d25-4aa3-9911-45dee3d15a63.png)

**Comment by lucas-zimerman** [comment]
would you mind giving it a try to the latest version of sentry/cli with the given command?
`yarn sentry-cli sourcemaps inject --org <organization> --project <project> ./www && sentry-cli sourcemaps upload --org sentry-sdks --project aspnet-core ./www`

There's no need to specify the release nor dist since the new cli has a better way of linking the sourcemaps.

Ideally that stacktrace shouldn't show as app:// if the source file exists on the dist folder, is this still happening and is this a debug or release build?

**Comment by sawaca96** [comment]
Oh, sorry...!

We finally know the cause.

According to my CI process, the android-related logic was uploading the source map to Sentry, but the ios-related logic was not uploading the source map to Sentry.

So, the problem was caused by the name of the dist file and the name of the source map not matching in the case of IOS.

I changed the release and dist to 0.0.1.androd 0.0.1.ios so that the source map was uploaded twice.

It works on ios too!
